numba/np/arraymath.py

```diff
@@ -704,75 +704,132 @@ def array_argmin_impl(arry):
     return impl_ret_untracked(context, builder, sig.return_type, res)
 
 
-@lower_builtin(np.argmax, types.Array)
-@lower_builtin("array.argmax", types.Array)
-def array_argmax(context, builder, sig, args):
-    ty = sig.args[0].dtype
+@register_jitable
```

Comment:
Note that these three functions (array_argmax_impl) are unchanged internally, they are just moved into top-level functions.

Reply:
thanks, comments like this help with review :)